### PR TITLE
Added a generic highlighting system

### PR DESCRIPTION
**Pull request by syc0n**
Adds Highlighting for a list defined in : src/ts/content/chat/ChatConfig.tsx
+ your own nickname 
later on this can be combined with a sound event

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/CUModSquad/cu-patcher-ui/pull/87" target="_blank">ReviewNinja

File: src/sass/_chat.scss [hunk 1]
```diff
@@ -383,6 +383,9 @@
               padding: 0px 5px;
             }
           }
+          .chat-room-highlight {
+            color: #FFE457;
```
Comment (Mehuge) [inline src/sass/_chat.scss]: This should probably be a variable in _constants.css

**Comment by Mehuge** [comment]
Looks good other than the one point raised.

File: src/sass/_chat.scss (hunk 1 above)
Comment (syc0n) [inline src/sass/_chat.scss]: for   .chat-room-link  I had it as constant but jb replaced it with :(
  .chat-room-link {
   color: #D4BD7F;

Comment (Mehuge) [inline src/sass/_chat.scss]: ok, gonna accept it as it is, can make it a constant later

Comment (Mehuge) [inline src/sass/_chat.scss]: !resolved

**Pull request merged**